cmd/poller/collector/collector.go

```diff
@@ -562,7 +548,7 @@ func (me *AbstractCollector) LoadPlugins(params *node.Node, c Collector, key str
 			me.Logger.Debug().Msgf("loaded built-in plugin [%s]", name)
 			// case 2: available as dynamic plugin
 		} else {
-			p = c.LoadPlugin(name, abc)
+			p = me.LoadPlugin(name, abc)
```

Comment:
Me and Rahul had tried this in last week, but it's not working. Because, me object don't have all the functions as c object, So while executing this line, it's not able to identify which custom plugin needs to be invoked. 

Before this change(all labels and metrics from plugin were populated):
```
hardikl@hardikl-mac-0 harvest % ./bin/poller --poller cluster-21 --loglevel 2 --promPort 12991 -c Zapi  -o Volume --config harvest.yml 
2:01PM INF ./poller.go:184 > log level used: info Poller=cluster-21
2:01PM INF ./poller.go:185 > options config: harvest.yml Poller=cluster-21
2:01PM INF ./poller.go:222 > started in foreground [pid=23529] Poller=cluster-21
2:01PM INF github.com/netapp/harvest/v2/cmd/poller/collector/helpers.go:123 > best-fit template [conf/zapi/cdot/9.8.0/volume.yaml] for [9.11.1] Poller=cluster-21 collector=Zapi:Volume
2:01PM INF ./poller.go:345 > Autosupport scheduled. Poller=cluster-21 asupSchedule=24h
2:01PM INF ./poller.go:354 > poller start-up complete Poller=cluster-21
2:01PM INF ./poller.go:502 > updated status, up collectors: 1 (of 1), up exporters: 1 (of 1) Poller=cluster-21
2:02PM WRN github.com/netapp/harvest/v2/cmd/collectors/zapi/plugins/volume/volume.go:113 > Failed to collect disk data error="no instances => no records found" Poller=cluster-21 object=Volume plugin=Zapi:Volume
2:05PM WRN github.com/netapp/harvest/v2/cmd/collectors/zapi/plugins/volume/volume.go:113 > Failed to collect disk data error="no instances => no records found" Poller=cluster-21 object=Volume plugin=Zapi:Volume
```

After this change(not able to invoke any code from volume.go plugin - no labels/metrics were populated):
```
hardikl@hardikl-mac-0 harvest % ./bin/poller --poller cluster-21 --loglevel 2 --promPort 12991 -c Zapi  -o Volume --config harvest.yml 
2:10PM INF ./poller.go:184 > log level used: info Poller=cluster-21
2:10PM INF ./poller.go:185 > options config: harvest.yml Poller=cluster-21
2:10PM INF ./poller.go:222 > started in foreground [pid=24457] Poller=cluster-21
2:10PM INF github.com/netapp/harvest/v2/cmd/poller/collector/helpers.go:123 > best-fit template [conf/zapi/cdot/9.8.0/volume.yaml] for [9.11.1] Poller=cluster-21 collector=Zapi:Volume
2:10PM INF ./poller.go:345 > Autosupport scheduled. Poller=cluster-21 asupSchedule=24h
2:10PM INF ./poller.go:354 > poller start-up complete Poller=cluster-21
2:10PM INF ./poller.go:502 > updated status, up collectors: 1 (of 1), up exporters: 1 (of 1) Poller=cluster-21
```